charts/redpanda-console-oauth2proxy/templates/_helpers.tpl

```diff
@@ -109,3 +109,11 @@ variable or return the imageRegistry as specified via the values.
     {{- printf "%s" $registryName -}}
 {{- end -}}
 {{- end -}}
+
+
+{{/*
+Expand the template to get the complete oauth2proxy image path.
+*/}}
+{{- define "oauth2proxy.image" -}}
+{{ .Values.oauth2Proxy.image.registry }}{{ .Values.oauth2Proxy.image.repository }}:{{ .Values.oauth2Proxy.image.tag }}
```

Comment:
I think you need to use printf like here https://github.com/trifork/cheetah-charts/blob/main/charts/redpanda-console-oauth2proxy/templates/_helpers.tpl#L38